src/Observable/fluctuations.jl

```diff
@@ -0,0 +1,71 @@
+# auxiliary function to compute occupation number fluctuations
+function compute_fluctuations(
+    file    :: HDF5.File,
+    Λ       :: Float64
+    ;
+    verbose :: Bool = true
+    )       :: Float64
+
+    # read symmetry 
+    symmetry = read(file["symmetry"])
+
+    # init number fluctuations
+    var = 0.0
+
+    if symmetry == "su2"
```

Comment:
Should we also move the implementations to a fluctuation_lib? would fit better to what we have done for all other symmetry dependent things

Reply:
Yes, that is probably better.